package.json

```diff
@@ -31,18 +32,21 @@
     "test:eslint": "eslint --ext=js --ext=jsx .",
     "test:stylelint": "stylelint \"**/*.css\"",
     "test:unit": "react-scripts test --env=jsdom",
-    "test": "npm run test:eslint && npm run test:stylelint && npm run test:unit",
+    "test:server": "mocha --recursive server/test/",
+    "test": "npm run test:eslint && npm run test:stylelint && yarn test:server && npm run test:unit",
```

Comment:
Can we have all of these use either npm or yarn instead of the inconsistency?

Reply:
Sure

Reply:
Actually I'll just fix it now